libs/ui/src/components/ArtCard/ArtCardBase.tsx

```diff
@@ -39,6 +39,13 @@ export const ArtCardBase: FC<ArtCardBaseProps> = ({
   } = useDisclosure()
 
   const [actionType, setActionType] = useState<ArtActionType>()
+  const [hover, setHover] = useState({ color: 'gray.100' })
+  const [color, setColor] = useState('white')
+
+  useEffect(() => {
+    setHover({ color: isLiked ? 'red.200' : 'gray.100' })
+    setColor(isLiked ? 'red.400' : 'white')
+  }, [])
```

Comment:
It seems the color is not being updated because you set the colors only at the first render. 
By the way, was the prior code not updating the color after a click? As far as I know,  As far as I know, a component needs to be re-rendered if the prop changes.

```suggestion
  }, [isLiked])
```